bin/proxy.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-console */
+require('babel-register');
+
+const http = require('http');
+
+const config = require('config');
+const httpProxy = require('http-proxy');
+
+
+const proxy = httpProxy.createProxyServer({ secure: false });
+const apiHost = config.get('proxyApiHost', null) || config.get('apiHost');
+const frontendHost = `http://${config.get('serverHost')}:${config.get('serverPort')}`;
+
+console.log(`apiHost: ${apiHost}`);
+console.log(`frontendHost: ${frontendHost}`);
+
+function getHost(req) {
+  if (!req.headers['user-agent'].includes('Android') || req.url.startsWith('/api/')) {
+    return apiHost;
+  }
+  return frontendHost;
+}
+
+const server = http.createServer((req, res) => {
+  console.log(`${req.method} ${req.url}`);
```

Comment:
Oh, I forgot to mention. This creates a lot of noise in the logs since we already see frontend server logging. Could this log message only go out for 4xx or 5xx responses?